utils/Cargo.toml

```diff
@@ -15,6 +15,8 @@ ark-serialize = { workspace = true, features = ["derive"] }
 async-trait = { workspace = true }
 clap = { workspace = true }
 committable = "0.2"
+derive_more = { workspace = true }
+dotenvy = "0.15.7"
```

Comment:
Is this needed? It seems there are no other changes to this crate.

Reply:
you're right, it's unused
https://github.com/EspressoSystems/espresso-network/commit/ec2cfaf48d106a8a1f25fa6d18e38c60f9cde5aa